eng/common/pipelines/templates/steps/docs-metadata-release.yml

```diff
@@ -21,6 +21,15 @@ parameters:
   CloseAfterOpenForTesting: false
 
 steps:
+- pwsh: |
+    if ($IsWindows) {
+      REG ADD HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\FileSystem /f /v LongPathsEnabled /t REG_DWORD /d 1
```

Comment:
I'm not a big fan of manually enabling long path on windows as it is pretty unreliable for general usage. Is there any way to fix the paths such that we don't hit the long file paths? If not then is there a way to make this template only run on non-windows?

Reply:
This isn't run on developer workstations so I think the blast radius of this change is contained. Enabling this flag in Windows is probably less painful then having someone stand up a Linux machine if they need to debug/test manually.

Reply:
Exactly this. The reason I'm submitting this PR is to make certain that nobody hits it in the future. One possible solution would be to swap the Java build phase to use `ubuntu`.

Reply:
I'm fine with this as a stop gap. Do we know if there is anyway to stop the long path files from being created to begin with?